lib/report.js

```diff
@@ -22,22 +22,46 @@ function jsonReport(errors) {
 function print(options, json) {
   if (options.reportOnOneLine) {
     if (json.type === 'review-errors') {
-      json.errors.forEach((errorForFile) => {
-        errorForFile.errors.forEach((error) => {
-          console.log(
-            JSON.stringify({
-              path: errorForFile.path,
-              ...error
+      if (json.errors.length > 0) {
+        return safeConsoleLog(
+          json.errors
+            .map((errorForFile) => {
+              return errorForFile.errors
+                .map((error) => {
+                  return JSON.stringify({
+                    path: errorForFile.path,
+                    ...error
+                  });
+                })
+                .join('\n');
             })
-          );
-        });
-      });
+            .join()
```

Comment:
```suggestion
            .join("\n")
```

This needs to be like this, otherwise the errors from different files will be joined by `,`

Reply:
Oh right, good catch!